src/main.rs

```diff
@@ -231,6 +231,14 @@ fn create_app<'a, 'b>(
                 .default_value(&api_server_path)
                 .group("vmm-config"),
         )
+        .arg(
+            Arg::with_name("restore-from")
```

Comment:
I tested that one (I did not before, shamelessly...) and it works.
I'd prefer if we call it `--restore` though, sorry for the nitpicking.

Reply:
Oh yeah no problem, and of course I tested it too :)